src/Fixer/PhpUnit/PhpUnitAttributesFixer.php

```diff
@@ -442,7 +442,9 @@ private static function fixDepends(Tokens $tokens, int $index, Annotation $annot
     private static function fixRequires(Tokens $tokens, int $index, Annotation $annotation): array
     {
         $matches = self::getMatches($annotation);
-        \assert(isset($matches[1]));
+        if (!isset($matches[1])) {
+            return [];
```

Comment:
so far, we claimed it will never happen with assertion (failing assertion)

now we allow that to happen and handle somehow.
but can that really happen? is that return ever executed? do we have a test case confirming that ?

Reply:
This is an early exit, few lines lower the condition `if (!isset($matches[2]) || !isset($map[$matches[1]])) {` will prevent going further anyway.

> is that return ever executed?

It is: https://coveralls.io/builds/69814192